dist/inverted-index.js

```diff
@@ -12,12 +12,14 @@ var _createClass = function () { function defineProperties(target, props) { for
 
```

Comment:
Good job with the transpiling of your code to ES6. For best practise sake, what do you think about having this file on your git repository? I feel it is not supposed to be pushed or what do you think since you already have the original code uploaded?